### ISSUE TITLE
Add support for switching accounts

### ISSUE THREAD

**Issue by miguelmich** [foxythemes/jira-cli#8]
Like @fluffels pointed out in #6, it would be great to have an option to setup multiple instances of JIRA and be able to switch between them, I suggest to add support inside the main config file because it's a better way to centralize the general settings.

**Comment by fluffels** [comment]
@miguelmich you're happy with the command itself on the #6 ? it's just the implementation you don't like?

**Comment by miguelmich** [comment]
@fluffels yes, I think there is a better way to handle multiple accounts, the main idea that I have in mind is to save an array of the current credentials object and then it will be easy to "activate" one of those instances like, for example, `jira config switch` and then display a prompt with the current options. 

Apart from this, the implementation that you did looks good, but it should be located in a method inside the config class and not inside the **jira.js** class which acts like some kind of "command router" between the commander index and the different classes.

The roadmap in order to make this new feature works could be:
1. Add a command to setup multiple accounts using the current config file, and set an "active" parameter which determinates the active instance
2. Modify the current init method in order to get the "active" instance and load it from the start
3. Add config commands to modify & remove the different instances
